protocol/messenger.go

```diff
@@ -2807,14 +2807,23 @@ func (r *ReceivedMessageState) addNewMessageNotification(m *common.Message) erro
 	}
 	contactID := contactIDFromPublicKey(pubKey)
 
-	r.Response.Notifications = append(
-		r.Response.Notifications,
-		MessageNotificationBody{
-			Message: m,
-			Contact: r.AllContacts[contactID],
-			Chat:    r.AllChats[m.ChatId],
-		},
-	)
+	chat := r.AllChats[m.ChatId]
```

Comment:
you should use straight away `localChatID`, not `ChatId` :)

Reply:
fixed